frontend/src/components/user-service/signup-page/SignupButton.tsx

```diff
@@ -0,0 +1,14 @@
+import { ArrowRightIcon } from "@radix-ui/react-icons";
+
+export default function SignupButton({onClick} : {onClick : ()=>void}) {
+    return (
+      <>
+        <div className="flex justify-center m-3"> 
+          <button onClick={onClick} className="border bg-orange-300 rounded-lg p-1 flex items-center">
+            <ArrowRightIcon/>
+            <div>Signup</div>
+          </button>
```

Comment:
- would be nice to use shadcn `<Button />` component instead of native `<button />`
- the arrow icon is right beside the signup text which doesn't look too great. would suggest adding more padding around the button and adding a gap between the button and text. something like
```tsx
<Button onClick={onClick} className="border bg-orange-300 hover:bg-orange-300/75 rounded-lg px-4 py-1.5 flex items-center gap-1">
  <ArrowRightIcon />
  <div>Sign up</div>
</Button>
```

Reply:
got it